app/src/main/java/com/otaman/notes/model/Note.kt

```diff
@@ -15,4 +17,30 @@ data class Note(
 
     @ColumnInfo (name = "content")
     val content: String
-)
\ No newline at end of file
+): Parcelable {
+    constructor(parcel: Parcel) : this(
+        parcel.readString()!!,
+        parcel.readString()!!,
+        parcel.readString()!!
+    )

```

Comment:
There is an easier way to create a `Parcelable` implementation using `kotlin-parcelize` plugin: https://developer.android.com/kotlin/parcelize